server/resolvers/helpers/retrieveCommands.js

```diff
@@ -0,0 +1,123 @@
+const commands = require('../../resources/commands.json');
```

Comment:
Would it be possible to rename commands to commandsV1? Otherwise it might give the impression that V1 is more authoritative or "normal" than V2 is, when V2 is actually the new standard going forward.

Reply:
Yep, I can make that change!

Reply:
So after looking into this, it does inflate the PR a bit. Like my comment in https://github.com/w3c/aria-at-app/pull/840#discussion_r1396413513, I'd rather to make this change in a follow up PR.